tests/v2/validation/provisioning/rke1/provisioning_cloud_provider_test.go

```diff
@@ -0,0 +1,103 @@
+//go:build (validation || extended) && !infra.any && !infra.aks && !infra.eks && !infra.rke2k3s && !infra.gke && !infra.rke1 && !cluster.any && !cluster.custom && !cluster.nodedriver && !sanity && !stress
```

Comment:
More a clarifying question - what is the expected frequency that we want for this test?

Reply:
we need to decide on this as a team, as I feel like right now we don't have consensus on when to add this. But I just wanted the same settings that our normal provisioning tests has for this test.